Cargo.toml

```diff
@@ -34,6 +34,7 @@ async-web-client = { version = "0.6.2", default-features = false }
 http = "1"
 blocking = "1.4.1"
 tower-service = { version = "0.3.3",  optional=true }
+actix-web = { version = "4", features = ["actix-tls", "rustls-0_23"], optional = true}
```

Comment:
move to dev dependencies and make it non-optional (I don't think cargo supports optional dev dependencies atm)